include/roboteam_ai/stp/new_skills/Kick.h

```diff
@@ -29,6 +29,9 @@ class Kick : public Skill {
      * Gets the skill name
      */
     const char *getName() override;
+
+    private:
+        int kickAttempts = 0;
```

Comment:
Since when do skills keep track of logic?

Reply:
Since this is the most efficient way to make sure we do a forced kick whenever we don't get ballSensor confirmation